azure-iot-device/azure/iot/device/provisioning/pipeline/provisioning_pipeline.py

```diff
@@ -29,6 +30,11 @@ def __init__(self, security_client):
         self.on_provisioning_pipeline_disconnected = None
         self.on_provisioning_pipeline_message_received = None
 
+        # if isinstance(security_client, X509SecurityClient):
+        #     stage_security = pipeline_stages_provisioning.X509SecurityClient()
+        # else:
+        #     stage_security = pipeline_stages_provisioning.UseSymmetricKeySecurityClient()
```

Comment:
I think you don't need this if you're combining both stages into one.  But, if you leave these as 2 stages, there's no harm in having both stages always be in the pipeline.  If one stage handles X509 and the other handles SK, you can keep them both,  Only one will be used in any specific registration, but the other one will not cause any problems.

Either way, I think you can remove this comment please.